data/mods/Xedra_Evolved/mutations/paraclesians/salamander_mutations.json

```diff
@@ -18,7 +18,7 @@
     "points": 2,
     "visibility": 4,
     "ugliness": 3,
-    "description": "Your skin has has magickal flames flaring out of it on occasion.  You can operate in warmer climates than normal humans but begin to take damage from getting wet.",
+    "description": "Your skin has magickal flames flaring out of it on occasion. You can operate in warmer climates than normal humans but begin to take damage from getting wet.",
```

Comment:
```suggestion
    "description": "Your skin has magical flames flaring out of it on occasion.  You can operate in warmer climates than normal humans but begin to take damage from getting wet.",
```

Reply:
This one I'm pretty sure is intended spelling within XE for otherworldly magical nonsense, but @GuardianDll might know better, if it is not intended there is quite a lot more to fix.

Reply:
I only spell magick with a k.  It's a choice.    The inspirational quote that led to this state of affairs was "Never trust anyone who spells magick with a 'k'. "